Add copy into rel table with multiple from-to pairs to data import se… (#441)
kuzudb/kuzu-docs: src/content/docs/import/csv.md

COPY Follows FROM "follows.csv" (SKIP=3);
```

### Relationship table with multiple `FROM-TO` pairs

> If a relationship table has multiple `FROM-TO` pairs, you need to specify which pair to insert through `from, to` options. For example

Reviewer: ```suggestion
If a relationship table has multiple `FROM-TO` pairs, you need to specify which pair to insert by specifying `from` and `to` parameters. For example:
```

```sql

> Copy Knows FROM 'knows_user_user.csv' (from='User', to='User');

Reviewer: ```suggestion
COPY Knows FROM 'knows_user_user.csv' (from='User', to='User');
```
